tests/web_api/test_tracking.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
```

Comment:
Renamed to `measure_tracking.py` to avoid automatic execution with other tests.
I don't know what the upper bound should be and if it should be different on circle. Any suggestions?